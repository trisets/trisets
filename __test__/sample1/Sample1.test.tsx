import Sample1 from '@/pages/sample1';
import { render } from '@testing-library/react';
import { expect, test } from 'vitest';

test('sample1', () => {
  const { container } = render(<Sample1 />);
  expect(container.firstChild).toMatchSnapshot();
});
