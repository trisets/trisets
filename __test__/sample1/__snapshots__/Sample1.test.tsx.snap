// Vitest Snapshot v1

exports[`sample1 1`] = `
<h1>
  Page01
</h1>
`;
